test/compilable/test23047.d

```diff
@@ -0,0 +1,13 @@
+/* REQUIRED_ARGS: -defaultlib= -c -O
+ */
+
+// https://issues.dlang.org/show_bug.cgi?id=23047
+
```

Comment:
You just removed the `version(D_SIMD)`. Tests are failing on non-SIMD compilations.
```suggestion
version(D_SIMD):
```

Reply:
oops...